lizmap/modules/lizmap/module.xml

```diff
@@ -36,6 +36,6 @@
         <class name="qgisLayerDbFieldsInfo" file="classes/qgisLayerDbFieldsInfo.class.php"/>
         <class name="qgisFormControlsInterface" file="classes/qgisFormControlsInterface.php"/>
         <class name="lizmapServices" file="classes/lizmapServices.class.php"/>
-        <namespacePathMap name="Lizmap" dir="lib"/>
+        <namespacePathMap name="Lizmap" dir="lib" suffix=".php"/>
```

Comment:
it should be useless. Is there an issue when it is missing? if yes, we should probably fix something into Jelix.